cudamapper/src/application_parameters.cpp

```diff
@@ -219,6 +220,11 @@ void ApplicationParameters::create_input_parsers(std::shared_ptr<io::FastaParser
         target_parser = io::create_kseq_fasta_parser(target_filepath, kmer_size + windows_size - 1);
     }
 
+    if ((query_parser->get_num_seqences() < 5 || target_parser->get_num_seqences() < 5) && !custom_filtering_parameter)
```

Comment:
Shouldn't we check based on a number of bases rather than number of sequences?

Reply:
Absolutely - can we extract that from the query/target parsers?

Reply:
Not directly, we would have to loop over sequences and check their lengths. This shouldn't be too costly as we can stop looping once we find enough bases